src/transformers/util.ts

```diff
@@ -473,3 +473,30 @@ export function shallowCopy<T>(x: T): T {
 }
 
 export const cloneAttribute = shallowCopy;
+
+/**
+ * Extracts a list of numbers from an attribute in a dataset that is expected
+ * to contain numeric values. Errors if the attribute is undefined for some
+ * records, or if the attribute's values parse to non-numeric.
+ *
+ * @param dataset The dataset to extract from.
+ * @param attribute The attribute containing numeric values.
+ * @returns A list of the attributes values parsed to numbers.
+ */
+export function extractNumericValues(
```

Comment:
The name of this function makes me think we're going to extract the numeric values and throw away the rest, as opposed to erroring if there are non-numeric values. Could we rename to something like `ensureNumericValues`/`convertToNumericValues`/`extractAttributeAsNumeric`